07.md

```diff
@@ -20,6 +20,7 @@ Aside from these two basic above, the following functions can also be implemente
 async window.nostr.getRelays(): { [url: string]: {read: boolean, write: boolean} } // returns a basic map of relay urls to relay policies
 async window.nostr.nip04.encrypt(pubkey, plaintext): string // returns ciphertext and iv as specified in nip-04
 async window.nostr.nip04.decrypt(pubkey, ciphertext): string // takes ciphertext and iv as specified in nip-04
+async window.nostr.nip26.createDelegation(parameters): string // takes delegator parameters and returns a delgation token using the stored private key
```

Comment:
parameters is too hand-wavy, let's make this more explicit, what needs to be passed to this function exactly?